### ISSUE TITLE
ECIP-1019

### ISSUE THREAD

**Issue by gagarin55** [ethereumproject/EIPs#33]
Have concern about this proposal. Many projects support ETC only because it is easy to integrate using well-known libs (web3j, web3.js, etc). Changing API only for changing doesn't bring any value. Developers will have to do double work.

**Comment by ghost** [comment]
I have to agree. I would like to see the metric because the naming convention has been confusing from day-one. However, I have worked with expanse and the others and changing the convention is a whole lot heartburn porting code. You could keep the backward compatibility for double the work but it would likely stick around for many years, if not forever like the "Suicide" OP code.

**Comment by elaineo** [comment]
@gagarin55 that is a good point. In theory someone could modify web3 libs, et al, but that is not a priority for anyone right now and seems like needless work

**Comment by ghost** [comment]
You could add middleware like expanse or add ETC to the existing module. https://www.npmjs.com/package/@expanse/web3 but yeah, another thing to support and monitor.

**Comment by elaineo** [comment]
I do support logo standardization, however. Logo discussion is probably not appropriate for an ECIP.
